data/map planets.txt

```diff
@@ -278,6 +290,18 @@ planet Baianus
 	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
 	government Uninhabited
 
+planet Bailey
+	attributes ccor mining
+	landscape land/mercury1
+	description `Bailey is a dead, nearly tidally locked world scarcely larger and more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Despite its inhospitable conditions, Bailey's bountiful crust has brought several mining expeditions to the planet.`
+	description `	Like aphids sucking the sap from a plant, semi-mobile mining stations strip and refine the minerals, a method originally developed by the Confederated Councils. Since their fall, the pirates of the region have maintained the stations - although with considerably less organization than what was upheld by the central government.`
+	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located on a repulsor-lift facility hovering a meter above Bailey's surface. Floating in place thanks to the lack of atmosphere, the planet slowly rotates beneath it while the facility is always on Bailey's dark side.`
```

Comment:
If we're saying that the planet is nearly tidally locked and not entirely tidally locked, then I imagine that should the spaceport remain in place, it would still take an extremely long amount of time for the spaceport to go from the dark side to the light side of the planet. The planet's orbital period is about 213 days. If we're saying that the planet is 99% tidally locked, then we're looking at something like 30 to 60 years for the rotation of the planet to become an issue for the spaceport. (Not sure on the specifics, but it'd definitely be some number of decades.) 

With that said, does it really make sense to have the spaceport be constantly repulsed above the planet? That seems to me like either it'd be a considerable energy drain to do that, or repulsor tech is so incredibly efficient that it doesn't matter. I'd almost say to change it so that the spaceport only lifts and moves itself once a year to keep itself on the dark side, but then I'm questioning the size of this spaceport and whether such a feat is possible.

We could easily avoid this scenario by simply stating that the planet is entirely tidally locked with its host star. Is the "almost tidally locked" point important enough to this planet to keep? (It might also make sense to move the planet closer to the system center, because its current distance is 705.3, which would put it between the orbits of Earth and Venus should it be in Sol, while the description makes it sound closer.)

Reply:
I think the 'nearly tidally locked' is important. The intention is that the process of the crust melting and re-solidifying makes the planet more worthwhile to put the effort into mining a dead world. Something along the lines of the volatility of the crust bringing mineral close to the surface.

The spaceport likely isn't small as I'm envisioning it as the center of the planetary mining efforts, expeditions are dispatched from the port and return with their hauls of ore. This is contrasted with Bailey having lighter gravity than Earth, being Mercury like it'd have a surface gravity around 0.38G.

An actual ground-station that periodically blasts off to reposition deeper into the dark side would work... even if it'd be expensive it wouldn't have to be annually, just however frequently it'd take for the station to be near sunrise from however far onto the dark side they move it. So if they move it to just after sunset each time that would be a few decades before it'd have to be moved again.

I can draft a new spaceport that's mobile... I wonder, would the spaceport being on gargantuan caterpillar treads make sense? A tank the size of a town rolling along the dark side at a snail's pace.

Reply:
If all the settlements themselves move, I could see the spaceport move like a NASA launch plaform or like London in the Mortal Engines books. Presumably a thousand years of development makes facilities of this size more efficient/sense.

~~Use my previous description suggestion?~~

Reply:
It could be that the spaceport is a series of disconnected buildings that move in unison. At that point it's not much different than a collection of ships getting up and movng, except the buildings are space-worthy.

Reply:
I think moving the planet closer is also in order if the idea is that it's so close to the star that one side is molten.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/8c3f6117-a1cc-4f18-a1aa-af8ed32f1e24)
This is Mercury at the same zoom level.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/27ad5391-58de-4373-adcc-2036785e687a)